hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestBase.java

```diff
@@ -235,9 +235,8 @@ private TestExecStartResultCallback executeCommandInDocker(String containerName,
     }
     int exitCode = dockerClient.inspectExecCmd(createCmdResponse.getId()).exec().getExitCode();
     LOG.info("Exit code for command : " + exitCode);
-    if (exitCode != 0) {
-      LOG.error("\n\n ###### Stdout #######\n" + callback.getStdout().toString());
-    }
+
+    LOG.error("\n\n ###### Stdout #######\n" + callback.getStdout().toString());
```

Comment:
Revert the logging changes?

Reply:
This are intended: these are changes replicated from https://github.com/apache/hudi/pull/6175